assets/wire/components/ItemDetails.tsx

```diff
@@ -82,6 +84,25 @@ function ItemDetails({
                             />
                         )
                     )}
+                    {allMedia == null ? null : allMedia
+                        .filter((mediaItem) => mediaItem.guid !== featureMedia?.guid) // Filter out feature media
+                        .map((data) => {
+                            return (
+                                data.type === 'picture' ? (
+                                    <ArticlePicture
+                                        picture={data}
+                                        isKilled={isKilled(item)}
+                                        isCustomRendition={isCustomRendition(featureMedia)}
```

Comment:
why is it using here `featureMedia` instead of `data` ?

Reply:
my bad